support/azure/azure-kubernetes/create-upgrade-delete/akscapacityerror.md

```diff
@@ -44,13 +44,13 @@ The `KubernetesAPICallFailed` error message indicates that the AKS cluster didn'
 
 ### Solution 1: Select a different region
 
-The easiest and quickest solution is to try to deploy to a different region that has enough capacity at the given time (for example, NorthEurope instead of WestEurope).
+The easiest and quickest solution is to try to deploy to a different region (for example, NorthEurope instead of WestEurope or UAENorth instead of QatarCentral). To find nearby regions, visit the [Azure Geographies page](https://azure.microsoft.com/explore/global-infrastructure/geographies/#overview).
 
 This approach might not be feasible if you already have existing resources in the requested region, but it's the preferred solution in a dev/test scenario.
 
 ### Solution 2: Try deploying a cluster that has different settings
 
-The underlays that host the AKS control planes have different allocation reservations. Therefore, the underlays for public AKS clusters might have more capacity than the overlays for private clusters. If you encounter the `AKSCapacityError` error when you try to create a private cluster, try to create a public cluster instead.
+The infrastructure that hosts AKS control planes have different allocation reservations. Therefore, AKS might have more capacity for public clusters than for private clusters. If you encounter the `AKSCapacityError` error when you try to create a private cluster, try to create a public cluster instead or vice versa.
```

Comment:
I know not your change but....

The infrastructures that hosts AKS managed clusters ...

Reply:
Fixed, thanks